locales/gu_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -70,10 +70,11 @@ msgid "%s%s%s Billion Searches"
 msgstr ""
 
 msgctxt "noresults"
-msgid ""
+msgid "કોઈ પરિણામ નહી"
```

Comment:
The `msgid` is defines the string that needs to be translated - the `""` looks empty but actually it's part of the `"%sClick here%s [...]"` string below. Can you remove the change on this line?